errors.go

```diff
@@ -1,6 +1,6 @@
 package slack
 
-import "github.com/nlopes/slack/internal/errorsx"
+import "github.com/acaloiaro/slack/internal/errorsx"
```

Comment:
@acaloiaro I think the PR is failing to build in Travis because of these import statement changes.

Reply:
Ah good catch; thanks. I never checked in on this after PRing. Will fix when I have the time.

Reply:
Thanks @gregmarr -- tests are passing now.

Reply:
You're welcome, and thanks for the fix.